flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -326,6 +326,26 @@ public void testDropCatalog() throws Exception {
         assertThat(executor.getNumExecuteSqlCalls(), is(1));
     }
 
+    @Test
+    public void testShowViews() throws Exception {
+        TestingExecutor executor =
+                new TestingExecutorBuilder()
+                        .setExecuteSqlConsumer(
+                                (ignored1, sql) -> {
+                                    SHOW_ROW.setField(0, "v1");
```

Comment:
We should add only condition return "v1" when the input `sql` equals to `show views`.

Reply:
I have changed testShowViews().